static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -1,86 +1,67 @@
+interface Props extends RouteComponentProps<{}, {}, {}> {}
 
-export default function FeedbackListPage({location}: Props) {
+export default function FeedbackListPage({}: Props) {
   const organization = useOrganization();
 
-  const query = useFeedbackListQueryParams({
-    location,
-    queryReferrer: 'feedback_list_page',
-  });
-  const {isLoading, isError, data, pageLinks} = useFetchFeedbackList({query}, {});
-
   return (
     <SentryDocumentTitle title={t(`Bug Reports`)} orgSlug={organization.slug}>
-      <Layout.Header>
-        <Layout.HeaderContent>
-          <Layout.Title>
-            {t('Bug Reports')}
-            <PageHeadingQuestionTooltip
-              title={t(
-                'Feedback submitted by users who experienced an error while using your application, including their name, email address, and any additional comments.'
-              )}
-              docsUrl="https://docs.sentry.io/product/user-feedback/"
-            />
-          </Layout.Title>
-        </Layout.HeaderContent>
-      </Layout.Header>
-      <PageFiltersContainer>
-        <Layout.Body>
-          <Layout.Main fullWidth>
-            <LayoutGap>
-              <PageFilterBar condensed>
-                <ProjectPageFilter resetParamsOnChange={['cursor']} />
-                <EnvironmentPageFilter resetParamsOnChange={['cursor']} />
-                <DatePageFilter alignDropdown="left" resetParamsOnChange={['cursor']} />
-              </PageFilterBar>
-              <FeedbackTable
-                data={data ?? []}
-                isError={isError}
-                isLoading={isLoading}
-                location={location}
+      <FullViewport>
+        <Layout.Header>
+          <Layout.HeaderContent>
+            <Layout.Title>
+              {t('Bug Reports')}
+              <PageHeadingQuestionTooltip
+                title={t(
+                  'Feedback submitted by users who experienced an error while using your application, including their name, email address, and any additional comments.'
+                )}
+                docsUrl="https://docs.sentry.io/product/user-feedback/"
               />
-            </LayoutGap>
-            <PaginationNoMargin
-              pageLinks={pageLinks}
-              onCursor={(cursor, path, searchQuery) => {
-                browserHistory.push({
-                  pathname: path,
-                  query: {...searchQuery, cursor},
-                });
-              }}
-            />
-          </Layout.Main>
-        </Layout.Body>
-      </PageFiltersContainer>
+            </Layout.Title>
+          </Layout.HeaderContent>
+        </Layout.Header>
+        <PageFiltersContainer>
+          <LayoutGrid>
+            <FeedbackFilters style={{gridArea: 'filters'}} />
+            <FeedbackSearch style={{gridArea: 'search'}} />
+            <FeedbackIndex style={{gridArea: 'list'}} />
+            <FeedbackDetails style={{gridArea: 'details'}} />
```

Comment:
Ah I see - I don't love having to pass down the `style` prop to a bunch of components for the grid area name, especially since the grid area names are pretty static, but it makes sense to colocate it with the template area definition. Nothing needs to be changed here, just commenting.

Reply:
I could probably do something so that I don't need gridArea here, just 4 items lining up into the right spots.... 

BUT! in `static/app/components/feedback/list/feedbackListItem.tsx` i've got a more complex grid going on, and tbh i really love it:
```
 grid-template-areas:
    'right user time'
    'right message message'
    'right icons icons';
```
Passing in `style={}` doesn't bother me since, in that file, i only wrote one `const Foo = styled('div')` component, and the html nesting is super clean. 

Another alternative is to use a Box/Flex component. That means no gridArea, and no need to defined a bunch of one-off `styled()` things (and name them)... but then there's still extra html nodes kicking around.